### PR TITLE
Add blog authors and other minor updates

### PR DESCRIPTION
**Pull request by jwilander**
* Added the blog authors with links to community.mattermost.com and GitHub to bottom of blog posts
* Added blog authors on blog list page
* Removed blog tags and added more categories

### CODE REVIEW

File: site/content/blog/2019-01-24-submitting-great-prs.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 ---
 title: "Submitting Great PRs"
 date: 2019-01-24T00:00:00-04:00
```
[the remark this answers is not in the document]
Comment (hanzei) [reply site/content/blog/2019-01-24-submitting-great-prs.md]: We have the data in the font matter. I don't see a point in having to sources of truth.

**Comment by jasonblais** [comment]
The categories take me to a 404 error page, e.g. https://developers.mattermost.com/blog/npm-v6/categories/announcement

Otherwise, great improvements here 👍

File: site/content/blog/2019-01-24-submitting-great-prs.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jwilander) [reply site/content/blog/2019-01-24-submitting-great-prs.md]: 0/5 but it is nice for sorting on my filesystem

**Comment by jwilander** [comment]
Hmm looks like the categories were broken already

**Comment by jwilander** [comment]
Odd they work fine locally for me

**Comment by jwilander** [comment]
@jasonblais I couldn't figure out why the categories weren't working so in the interim I just replaced them in the sidebar with links to other blog posts

File: site/content/blog/2019-01-24-submitting-great-prs.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (lieut-data) [reply site/content/blog/2019-01-24-submitting-great-prs.md]: Agree on the filesystem sorting -- is it feasible to build the URL to access same without the date?

[the remark this answers is not in the document]
Comment (jwilander) [reply site/content/blog/2019-01-24-submitting-great-prs.md]: Looks like we can https://gohugo.io/content-management/organization/#override-destination-paths-via-front-matter

File: site/content/blog/2019-01-24-submitting-great-prs.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
 ---
 title: "Submitting Great PRs"
+slug: submitting-great-prs
```
[the remark this answers is not in the document]
Comment (jwilander) [reply site/content/blog/2019-01-24-submitting-great-prs.md]: Will add an alias for it

**Comment by jwilander** [comment]
Chatted with Jason about the sidebar, it's not perfect but we're ok with it for now

**Pull request merged**